sidebar/sidebar.html

```diff
@@ -53,6 +53,12 @@
         <label class="i18n">toolbar.output</label>
         <input name="output" type="text" class="w-100" autocomplete="off">
       </div>
+      <div class="form-group">
+        <select name="outputType" class="custom-select">
+          <option value="0" selected="" class="i18n">outputType.text</option>
+          <option value="1" class="i18n">outputType.function</option>
+        </select>
+      </div>
```

Comment:
You could add the warning also here, below the output type, preferably shown only if "function" is selected.